### PR TITLE
APP-1476: Make drop down select focus on top result

### PR DESCRIPTION
**Pull request by anjinai**
**20230327_Make drop down select focus on top result**

**Items to Note:** 

- Updates select and multi-select components to have the top (ordered) result already in focus such that a user can press enter and select that first item.
- PR includes tests for this behavior as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime",
-  "version": "0.1.4",
+  "version": "0.1.5",
```
Comment (mcous) [inline package.json:3]: What's the bump/release strategy for this library?

**Review by mcous — APPROVED** [review]
Tested behavior manually, and it works as advertised. Tests look good, too!

File: package.json (hunk 1 above)
Comment (anjinai) [reply package.json:3]: I think we’re trying to follow semver.  Anytime I make a bug fix/minor change to a component, I increase the patch version in prime and then go to app and update the version of prime that app uses.

**Comment by anjinai** [comment]
> Tested behavior manually, and it works as advertised. Tests look good, too!

Thanks for reviewing!

**Review by ehhong — APPROVED** [review]

**Pull request merged**